internal/pkg/print/pretty/pretty.go

```diff
@@ -52,7 +52,11 @@ const (
 			{{- printf "\n" -}}
 			{{- range . }}
 				{{ printf "output.%s" .Name | colorize "\033[36m" }}
-				{{ tostring .Description | trimSuffix "\n" | default "n/a" | colorize "\033[90m" }}
+				{{- if $.Module.Options.OutputValues -}}
```

Comment:
Please take a look at my comment below regarding `Settings` and `Options` and not exporting `Module.Options` and use `$.Settings.OutputValues` here.